### PR TITLE
fixed typo mistakes

### PR DESCRIPTION
**Pull request by toir427**
fixed typo mistakes

### CODE REVIEW

File: docs/guide/test-overview.md [hunk 1]
```diff
@@ -61,10 +61,10 @@ There's nothing wrong in creating tests covering behavior of existing implementa
 
 In some cases any form of automated testing could be overkill:
 
-- Project is simple and isn't getting anymore complex.
+- Project is simple and isn't getting any more complex.
 - It's a one-time project that will no longer be worked on.
 
-Still if you have time it's good to automate testing in these cases as well.
+Still if you have timed it's good to automate testing in these cases as well.
```
Comment (bizley) [inline docs/guide/test-overview.md:67]: Here it was ok

Comment (samdark) [reply docs/guide/test-overview.md:67]:
```suggestion
Still if you have time it's good to automate testing in these cases as well.
```

File: docs/guide/tutorial-i18n.md [hunk 2]
```diff
@@ -449,7 +449,7 @@ locale you are translating to:
 ```php
 $n = 3;
 echo Yii::t('app', 'You are the {n,selectordinal,one{#st} two{#nd} few{#rd} other{#th}} visitor', ['n' => $n]);
-// For English it outputs:
+// For English, it outputs:
```
[the remark this answers is not in the document]
Comment (samdark) [reply docs/guide/tutorial-i18n.md:452]:
```suggestion
// For English it outputs:
```

**Comment by samdark** [comment]
Thanks.

**Pull request merged**